### PR TITLE
Feature/11783 dates in custom fields without alerts

### PR DESCRIPTION
**Pull request by mujtaba7vals**
http://pm.7vals.com/issues/11783
Server side changes:
https://github.com/7Vals/ezofficeinventory/pull/10890/files

### CODE REVIEW

**Comment by mujtaba7vals** [comment]
cycle 0

File: app/models/custom_attributes/date_time_value.rb [hunk 1]
```diff
@@ -9,7 +10,11 @@ def value=(newVal)
     end
 
     def value
-      self.date_time_value
+      format_date_input(date_time_value, Company.find(company_id))
```
Comment (Anmol79) [inline app/models/custom_attributes/date_time_value.rb]: gem should not be dependant on EZO - write a method here to format the date with the format passed as in argument

File: app/views/custom_attributes/date_time_values/_edit_date_time_value.html.erb [hunk 2]
```diff
@@ -20,12 +22,23 @@
...
-  <div class="custom-attributes field datetime <%=cssClasses[:field]%>" style="display: none">
-    <input type="text" class="custom-attributes field" value="<%= date_time_value.value %>" required=<%= required %>/>
+  <div class="custom-attributes field datetime <%= cssClasses[:field] %>" data-date-field="true" style="display: none">
+    <%= text_field_tag '', format_date_input(date_time_value.date_time_value), class: 'custom-attributes field', readonly: true %>
   </div>
 </div>
 <% # END Inline Editing %>
+<% end %>
+
+<% unless request_from_webstore?(request) %>
```
Comment (Anmol79) [inline app/views/custom_attributes/date_time_values/_edit_date_time_value.html.erb]: get this in locals from ezo side

**Comment by Anmol79** [comment]
Cycle 1

**Pull request merged**